### PR TITLE
feat: Added animations to success views

### PR DESCRIPTION
**Pull request by daviroo**

### CODE REVIEW

**Review by Extheoisah — DISMISSED** [review]

File: app/screens/receive-bitcoin-screen/receive-btc.tsx [hunk 1]
```diff
@@ -527,76 +532,103 @@ const ReceiveBtc = () => {
...
+                </Pressable>
               </View>
-            </Pressable>
-          </View>
-        </View>
+            </View>
+          </>
+        )}
+        {state === PaymentRequestState.Paid && (
+          <>
```
Comment (nicolasburtey) [inline app/screens/receive-bitcoin-screen/receive-btc.tsx:613]: no need for `<>`?

File: app/screens/receive-bitcoin-screen/receive-usd.tsx [hunk 2]
```diff
@@ -496,6 +506,26 @@ const ReceiveUsd = () => {
             />
           </>
         )}
+        {state === PaymentRequestState.Paid && (
+          <>
```
Comment (nicolasburtey) [inline app/screens/receive-bitcoin-screen/receive-usd.tsx:510]: no need for <>?

File: app/screens/send-bitcoin-screen/send-bitcoin-success-screen.tsx [hunk 3]
```diff
@@ -46,11 +48,14 @@ const SendBitcoinSuccessScreen = ({
       contentContainerStyle={styles.contentContainer}
     >
       <View style={styles.Container}>
-        <GaloyIcon name={"payment-success"} size={128} />
-
-        <Text {...testProps("Success Text")} style={styles.successText}>
-          {LL.SendBitcoinScreen.success()}
-        </Text>
+        <Animated.View entering={PinwheelIn.duration(3000).springify().delay(50)}>
```
Comment (nicolasburtey) [inline app/screens/send-bitcoin-screen/send-bitcoin-success-screen.tsx:51]: what is the `3000` and `50` for? those magic value should be set as const to be documented.

Comment (daviroo) [reply app/screens/send-bitcoin-screen/send-bitcoin-success-screen.tsx]: I've refactored it into a component to keep the config in one place.

**Review by Extheoisah — APPROVED** [review]

**Pull request merged**